hackerRank/simple-array-sum/simple-array-sum.test.js

```diff
@@ -0,0 +1,10 @@
+const simpleArraySum = require('./simple-array-sum');
+const testCaseFileOne = require('./testCaseFileOne');
+
+test('adds 1 + 2 + 3 + 4 + 10 + 11  to equal 31', () => {
```

Comment:
Lets update this description to something better "sums an array for integers" maybe,but something more descriptive

Reply:
Done